src/pretix/control/forms/orders.py

```diff
@@ -13,7 +13,10 @@ class Meta:
         model = Order
         fields = ['expires']
         widgets = {
-            'expires': forms.DateInput(attrs={'class': 'datepickerfield'}),
+            'expires': forms.DateInput(attrs={
+                'class': 'datepickerfield',
+                'data-payment-weekdays-disabled': '[]'
```

Comment:
As this will *always* have this value, I again find the name misleading. How about ``data-is-payment-date=true`` instead to indicate it *is* a payment date? Because this is what this does, all other logic is in the ``body`` attribute.